notification-server/.env.template

```
@@ -0,0 +1,2 @@
+PGPASSWORD=$DB_PASSWORD
+DB_CONNECTION="host=localhost dbname=kpi-tracker user=postgres password=$DB_PASSWORD port=5432"
```

Why two different names for the variables here? Also I think it would be nice to prefix environment variables with something that identifies the service (we do this for other services)

```suggestion
CCD_NOTIFICATIONS_DB_PASSWORD=$CCD_NOTIFICATIONS_DB_PASSWORD
CCD_NOTIFICATIONS_DB_CONNECTION="host=localhost dbname=kpi-tracker user=postgres password=$CCD_NOTIFICATIONS_DB_PASSWORD port=5432"
```

and then align correspondingly everywhere else as well

---

I did it deliberately. 

When using localhost you would like to be able to reuse the existing connection. 

For other deployments it does not matter whether we do one or another.

Since we do it other places I can accept it, and the benefit is also minor.